tests/my_test.py

```diff
@@ -2,27 +2,32 @@
 import unittest
 import requests
 
-
 class YahooAPITestCase(unittest.TestCase):
     def test_for_successful_response(self):
         result = requests.get("http://www.yahoo.com")
         self.assertEqual(200, result.status_code)
         # OR
         self.assertTrue('OK' == result.reason)
 
-
 class CareerPortalTests(unittest.TestCase):
     def setUp(self) -> None:
         self.base_url = 'https://recruit-portnov.herokuapp.com/recruit/api/v1'
         self.headers = {'user-agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36'}
 
     def test_login(self):
-        positions = requests.get(self.base_url + '/positions')
+        positions = self.get_positions()
         json_positions = json.loads(positions.text)
-
         self.assertEqual(len(json_positions), 5)
 
-        result = requests.post(self.base_url + '/login', json={"email": "student@example.com", "password": "welcome"})
+        aid = 1
+        application = self.get_application(aid)
+        json_application = json.loads(application.text)
+        self.assertEqual(aid, json_application['id'])
+
+        aemail = 'student@example.com'
+        apassword = 'welcome'
```

Comment:
I like that you assigned all of your input data to variables. 
I would recommend taking it a step further, and moving all assignments of test data to the top of the test. It makes it easier to find and modify if there is eve a need.